extensions/resteasy-reactive/quarkus-resteasy-reactive-servlet/runtime/src/main/java/io/quarkus/resteasy/reactive/server/servlet/runtime/ServletRequestContext.java

```diff
@@ -574,4 +575,26 @@ public V setValue(V value) {
             return old;
         }
     }
+
+    @Override
+    public ServerHttpResponse sendFile(String path, long offset, long length) {
+        context.response().sendFile(path, offset, length);
+        return this;
+    }
+
+    @Override
+    public boolean isWriteQueueFull() {
```

Comment:
isWriteQueueFull and addDrainHandler don't really make sense here. The interface really needs some Javadoc but at the moment write()  is assuming you can write once, then you need to wait for a notification.

Reply:
Well, I honestly didn't know how to implement those. Any hint? I did need them for the vertx async part.